tests/api/test_committee_meetings.py

```diff
@@ -9,12 +9,22 @@ def setUp(self):
         self.fx = dbfixture.data(HouseData, CommitteeData, CommitteeMeetingData)
         self.fx.setup()
 
+    def tearDown(self):
+        self.fx.teardown()
+        super(TestCommitteeMeetingsAPI, self).tearDown()
+
     def test_premium_committee_meeting(self):
-        res = self.client.get('/committee-meeting/%s/' % self.fx.CommitteeMeetingData.premium_recent.id, base_url='http://api.pmg.test:5000/')
```

Comment:
Sorry about these unintended changes - I have python black installed in emacs now  and I forget to turn it off. It it gets in the way of understanding the PR and ignoring whitesapce in the PR doesn't work, we can clean it up.

For now I think it's ok.